Sources/PackageModel/SwiftSDKs/SwiftSDK.swift

```diff
@@ -617,11 +617,11 @@ public struct SwiftSDK: Equatable {
     ///
     /// - SeeAlso: ``sdkPlatformPaths(for:environment:)``
     public struct PlatformPaths {
```

Comment:
I think this may be technically ABI-breaking?

Reply:
Most likely, but I don't think SwiftPM's libraries are considered ABI stable?

Reply:
DMed.